modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js

```diff
@@ -15,6 +15,8 @@ import React from 'react';
 
 import TrafficSources from '../../../src/main/resources/META-INF/resources/js/components/TrafficSources';
 
+const noop = () => {};
```

Comment:
I also wouldn't worry about it in tests, unless you think it helps readability (which is the most important thing in tests anyway; you don't have to worry about re-renders in the context of a test suite because most `it` blocks render exactly once).

Reply:
Well, I replace all occurrences... OCD. My fault.